src/cobra/flux_analysis/fastcc.py

```diff
@@ -33,6 +35,12 @@ def _find_sparse_mode(
         The list of reactions to consider as consistent.
 
     """
+    warn("As of 2021, release V... this function contains a known bug "
+         "which allows reactions with zero net flux to be considered active,"
+         " as well as known discrepancies between this function "
+         "(the cobrapy implementation) and its matlab implementation "
+         "present in the cobra toolbox, "
+         "see: https://github.com/opencobra/cobrapy/issues/1154")
```

Comment:
Can't properly line wrap in this comment.

```suggestion
    warn("This function contains a known bug "
         "which allows reactions with zero net flux to be considered active,"
         " as well as known discrepancies between this function "
         "and its Matlab implementation "
         "present in the COBRA toolbox. "
         "Please, see https://github.com/opencobra/cobrapy/issues/1154 for more information.")
```